.github/labeler.yml

```diff
@@ -55,6 +55,22 @@ component/tasklist:
           - 'tasklist/**'
           - 'tasklist.Dockerfile'
           - 'webapps-common/**'
+component/optimize:
+  - changed-files:
+      - any-glob-to-any-file:
+        - '.ci/docker/test/*'
```

Comment:
FYI: We don't have any optimize files in that `.ci/` directory. Maybe this line is not necessary?

Reply:
maybe not, I just follow the list of dirs/files from Operate. I can see that in the `.ci` operate is most likely looking at `operate-docker-labels.golden.json` file. Is optimize going to have similar file there? if the answer is yes then we need to leave the config the way I made it now, otherwise we can remove it